draft-ietf-httpbis-semantics-latest.xml

```diff
@@ -1711,6 +1711,78 @@ deployed and not likely to be registered in a timely manner otherwise.</t>
 </section>
 </section>
 
+<section title="Trailer Fields" anchor="trailer.fields">
+   <iref item="trailer fields"/>
+   <iref item="trailers"/>
+
+<section title="Purpose" anchor="trailers.purpose">
+<t>
+   In some cases specific to the transfer coding or HTTP version, additional
```

Comment:
"In some HTTP versions, additional..."

or

"In some HTTP versions (e.g., using transfer codings in HTTP/1.1)..."

Reply:
yep, first alternative is better because next sentence uses chunked as example.